spec/integration/analytics_data_spec.rb

```diff
@@ -1,12 +1,11 @@
-require 'integration_test_helper'
+require 'spec_helper'
 
-class AnalyticsDataTest < IntegrationTest
-  def setup
-    super
+RSpec.describe 'AnalyticsDataTest', tags: ['integration'] do
```

Comment:
It would be good to change these to constants if we can (`describe AnalyticsData`)